config/samples/ccruntime/default/kustomization.yaml

```diff
@@ -10,8 +10,8 @@ images:
 - name: quay.io/confidential-containers/container-engine-for-cc-payload
   newTag: 98a790e8abdcc06c4b629b290ebaa217bf82e305
 - name: quay.io/confidential-containers/runtime-payload
-  newName: quay.io/confidential-containers/runtime-payload
-  newTag: kata-containers-338e18e4fd46756643cb1a307b6a20f24c350ad4
+  newName: quay.io/confidential-containers/runtime-payload-ci
+  newTag: kata-containers-d742fe37a92a7be600f2fb2d676108697b91abad-s390x
```

Comment:
Hey, please, drop the `-s390x` bit from here, otherwise it'll pull `-s390x` payload for `-amd64` hosts.
Using without defining the arch should be good enough, as the image is a multi-manifest one.

Reply:
Yep, it is gonna removed in the end. I am currently working on getting the CI job http://jenkins.katacontainers.io/job/confidential-containers-operator-main-ubuntu-20.04-s390x-SE-daily/ green (1st establishment). At the moment, there is no artifact for [`cc-se-image`](https://github.com/kata-containers/kata-containers/blob/CCv0/.github/workflows/cc-payload-after-push-s390x.yaml#L19) in the `runtime-payload`. When the establishment is finished, I will make a separate PR for the payload and make this use the default config. Thanks. 😉